wp/lib/bap_wp/src/z3_utils.ml

```diff
@@ -111,3 +111,59 @@ let mk_smtlib2_single (env : Env.t) (smt_post : string) : Constr.t =
   let ctx = Env.get_context env in
   mk_smtlib2 ctx smt_post decl_syms
 
+let construct_pointer_constraint (l: string list) (env1 : Env.t)
+    (env2: Env.t option): Constr.t =
+  let stack_end = Env.get_stack_end env1 in
+  let ctx = Env.get_context env1 in
+  let arch = match Env.get_arch env1 |> Bap.Std.Arch.addr_size with
+    | `r32 -> 32
+    | `r64 -> 64 in
+  let rsp = Env.get_rsp_name env1 in
+  let err_msg_input = "invalid register name in pointer register list" in
```

Comment:
Probably a good idea to have the name of the function in the error message, in my experience.

Reply:
Like `"construct_pointer_constraint: invalid register name in pointer register list"`